doc.go

```diff
@@ -990,5 +990,25 @@ that should not make it to production.
 	}
 
 	validate.Struct(t) // this will panic
+
+Non standard validators
```

Comment:
If you could add a note here about what Non standard validations are and that you have to register them manually that would be amazing :)